@navikt/core/react/src/typography/ErrorMessage.tsx

```diff
@@ -16,6 +17,11 @@ export interface ErrorMessageProps
    * Error text.
    */
   children: React.ReactNode;
+
+  /**
+   * Render a triangular warning icon.
+   */
+  icon?: boolean;
```

Comment:
We often use the `icon`-propname to indicate that you provide the icon yourself. Since this is just a boolean, could `showIcon`  be more fitting?